webpack.config.ts

```diff
@@ -5,14 +5,15 @@ import * as path from 'path'
 import * as webpack from 'webpack'
 
 const isProduction = process.argv.indexOf('-p') >= 0
+const isContinuousIntegration = process.argv.indexOf('--ci') >= 0
 const transpileOnly = process.argv.indexOf('-t') >= 0
 const sourcePath = path.join(__dirname, './src')
 const outPath = path.join(__dirname, './dist')
 
 const config: webpack.Configuration = {
   mode: isProduction ? 'production' : 'development',
   context: sourcePath,
-  devtool: isProduction ? 'source-map' : 'inline-source-map',
+  devtool: isContinuousIntegration ? false : isProduction ? 'source-map' : 'eval-source-map',
```

Comment:
We don't need source maps on CI, resulting in a much faster webpack build (~80s to ~24s).